### PR TITLE
change sentry to capture exception with message

### PR DESCRIPTION
**Pull request by s-block**

### CODE REVIEW

**Comment by s-block** [comment]
retest this please

**Comment by s-block** [comment]
retest this please

**Comment by s-block** [comment]
retest this please

File: cla_public/libs/utils.py [hunk 1]
```diff
@@ -43,3 +43,8 @@ def recursive_dict_update(orig, new):
         else:
             orig[key] = new[key]
     return orig
+
+
+def log_to_sentry(message):
+    if hasattr(current_app, 'sentry'):
+        current_app.sentry.captureMessage(message)
```
Comment (andyhd) [inline cla_public/libs/utils.py]: Is it worth changing this to a try/catch and logging normally if there's no sentry? Eg:
```python
try:
    current_app.sentry.captureMessage(message)
except SomeException:
    log.warning(message)
```

**Comment by s-block** [comment]
retest this please

**Pull request merged**